vllm/utils/flashinfer.py

```diff
@@ -502,8 +507,11 @@ def flashinfer_scaled_fp8_mm(
 
 @functools.cache
 def flashinfer_disable_q_quantization() -> bool:
-    """Cache result which only depends on the environment"""
-    return envs.VLLM_FLASHINFER_DISABLE_Q_QUANTIZATION
+    """Cache result which only depends on the attention config"""
```

Comment:
I don't _think_ we need to cache this anymore. The caching was because reading the environment was expensive